### PR TITLE
update plugins and dependencies

### PR DESCRIPTION
**Pull request by BCaxelbecker**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -248,12 +271,12 @@
...
 				<groupId>org.apache.httpcomponents</groupId>
 				<artifactId>httpclient</artifactId>
-				<version>4.5.1</version>
+				<version>4.5.6</version>
 			</dependency>
 			<dependency>
 				<groupId>junit</groupId>
 				<artifactId>junit</artifactId>
-				<version>4.12</version>
+				<version>4.13-beta-1</version>
```
Comment (hkernbach) [inline pom.xml:279]: beta version is fine?

Comment (BCaxelbecker) [reply pom.xml:279]: mvn versions:display-dependency-updates shows it as the last release version

Comment (hkernbach) [reply pom.xml:279]: image

Comment (hkernbach) [reply pom.xml:279]: Building works, but I get: 

```
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.283 s - in com.arangodb.spark.java.ArangoSparkJavaWriteTest
19/01/18 15:08:39 INFO ShutdownHookManager: Shutdown hook called
19/01/18 15:08:39 INFO ShutdownHookManager: Deleting directory /private/var/folders/hd/z2qv4td92tv2fxgh8y8pxfwr0000gn/T/spark-c72e4e71-09c7-4c83-b626-be77945d4a97
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   ArangoSparkJavaReadTest.loadAll:59 
Expected: is <100L>
     but: was <0L>
[ERROR]   ArangoSparkJavaReadTest.loadAllWithHTTP:72 
Expected: is <100L>
     but: was <0L>
[ERROR]   ArangoSparkJavaReadTest.loadAllWithLoadBalancing:78 
Expected: is <100L>
     but: was <0L>
[ERROR]   ArangoSparkJavaReadTest.loadWithFilterStatement:66 
Expected: is <50L>
     but: was <0L>
```

when running the tests.

Comment (hkernbach) [reply pom.xml:279]: tested with devel. with 3.4 its working

**Review by hkernbach — APPROVED** [review]
LGTM

**Pull request merged**